src/main/java/org/takes/misc/VerboseList.java

```diff
@@ -35,6 +35,7 @@
  * @author I. Sokolov (happy.neko@gmail.com)
  * @version $Id$
  * @param <T> Type of item
+ * @since 0.1
```

Comment:
@xupyprmv it was developed very recently, please see https://github.com/yegor256/takes/commits/48864f1bc7e922c42ecd0d5ed283359182bbf3fa/src/main/java/org/takes/misc/VerboseList.java